spring-petclinic-api-gateway/src/main/java/org/springframework/samples/petclinic/api/application/OwnerDetails.java

```diff
@@ -27,19 +28,20 @@
  * @author Maciej Szarlinski
  */
 @Data
+@NoArgsConstructor
```

Comment:
Why do we need no-args constructor?

Reply:
I don't remember why I have to add the @NoArgsConstructor annotation. I will try to remove it.